gensim/models/ldamodel.py

```diff
@@ -193,7 +195,7 @@ def get_Elogbeta(self):
 # endclass LdaState
 
 
-class LdaModel(interfaces.TransformationABC):
+class LdaModel(interfaces.TransformationABC,basemodel.BaseTopicModel):
```

Comment:
slight code style issue. Should be `interfaces.TransformationABC, basemodel.BaseTopicModel`.